scripts/gitlab_trigger_client_publish

```diff
@@ -19,16 +30,29 @@ if [ ! -d $WORKSPACE/integration ]; then
   exit 1
 fi
 
+# Get the integration versions to publish
+if [ "$trigger_from_repo" == "meta-mender" ]; then
+  # Special handling on meta-mender: publish master + last three releases
+  cd $WORKSPACE/integration
+  remote=$(git config -l | \
+           sed -n -E 's|^remote\.([^.]+)\.url=.*github\.com[/:]mendersoftware/integration(\.git)?$|\1|p')
+  integration_versions=$(git for-each-ref --sort=-creatordate --format='%(refname:short)' 'refs/tags' | \
+                         sed -E '/(^[0-9]+\.[0-9]+)\.[0-9]+$/!d;s//\1.x/' | \
+                         uniq | \
+                         head -n 3)
```

Comment:
Suggest making the `3` a variable like `NUMBER_OF_MINOR_VERSIONS` and put it on top. I can imagine we might want to change this at some point.

Reply:
Yes! Fixing